handlers.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/fullstorydev/grpcurl"
 )
 
+var emitDefaults = true
+
```

Comment:
This seems bad... I think you need to do something like:
```go
	result := rpcResult{
		descSource:   descSource,
		emitDefaults: options.EmitDefaults,
		Requests:     &reqStats,
	}
```
around line 160.  Then pass it through as a parameter to toRpcError / responseToJSON